source/install/install-kubernetes-mattermost.rst

```diff
@@ -61,7 +78,98 @@ Save the file as ``mattermost-installation.yaml``:
       username: ""
       password: ""
 
-**3. Edit the Installation Manifest File**
+Save the file as ``mattermost-installation.yaml``.
+
+**3. Apply the Installation Manifest File**
+
+To initiate deployment, apply the file, specifying the correct path, using:
+
+.. code-block:: sh
+
+  $ kubectl create ns mattermost
+  $ kubectl apply -n mattermost -f /path/to/mattermost-installation.yaml
+
+The deployment process can be monitored in the Kubernetes user interface.
+
+**4. Configure DNS and Use Mattermost**
+
+When the deployment is complete, obtain the hostname or IP address of your Mattermost deployment using the following command:
+
+.. code-block:: sh
+
+  $ kubectl -n mattermost get ingress
+
+Copy the resulting hostname or IP address from the ``ADDRESS`` column, open your browser, and connect to Mattermost.
+
+Use your domain registration service to create a canonical name or IP address record for the ``ingressName`` in your manifest,
+pointing to the address you just copied. For example, on AWS you would do this within a hosted zone in Route53.
+
+Navigate to the ``ingressName`` URL in your browser and use Mattermost.
+
+Restoring an Existing Mattermost MySQL Database
+-----------------------------------------------
+
+You can leverage the Mattermost Operator to utilize an existing Mattermost MySQL database with a new Mattermost installation, in its own namespace.
+The steps you follow to create and upload your backup depends on the provider you're using and your use case. It's
+recommended that you consult the relevant documentation or, if your deployment is managed in a different way, consult your Administrator.
+
+It is important to note that this process requires the creation of a new Mattermost
+installation - editing the existing ``.yaml`` files is not recommended and can result in data loss.
+
+The process described below needs to be completed prior to proceeding with the Mattermost deployment.
+
+1. Create a backup of your database (e.g., using *mysqldump*).
+2. Deploy a new server (e.g., an AWS instance).
+3. Install a backup program and back up the database on the new server/instance.
+4. Upload the backed up database to your cloud storage provider (e.g., Amazon S3).
+5. Create a ``secret.yaml``` file:
+
+Open a text editor and create a text file containing your credentials which will be used to access the uploaded database.
+
+Save the file as ``secret.yaml``. The example below is for AWS/S3.
+
+.. code-block:: yaml
+
+   apiVersion: v1
+   kind: Secret
+   metadata:
+    name: test-restore
```

Comment:
This is a "freestanding" file so the name doesn't seem to be used in any of the following examples. Correct?